preTrainedSample.py

```diff
@@ -418,10 +442,9 @@ def loadModel(fName):
     modelTrained = trainModel(samplesTrain, labelsTrain, samplesValidate,
                               labelsValidate, vectorizer, modelArch,
                               batchSize=batchSize, epochs=epochs)
-    testFileName = "/Users/lance/Documents/GitHub/DogWhistleTF/testProbes.txt"
-    results = testModel(testFileName, modelTrained, classLabels)
+    testFileName = r"C:\Users\desmo\OneDrive\Desktop\GitHub\DogWhistleTF\testProbes.txt"
```

Comment:
let's use a list of file names that allows us to choose our own file using an index. Something like the dataSource definition above.
testFileName = ['/Users/lance/Documents/GitHub/DogWhistleTF/testProbes.txt', r"C:\Users\dsm84762\PycharmProjects\FinalDW\DogWhistleTF\testProbes.txt"][1]

Reply:
I'll add a function that sets the path based on the OS of the user.